packages/base-driver/lib/basedriver/arguments.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash';
+import { validateCaps } from './capabilities';
+
+function parseKnownArgs (serverArgs, knownArgNames, argsConstraints) {
```

Comment:
it looks like our knownArgNames are always equal to keys of argsConstraints, which makes this function simpler